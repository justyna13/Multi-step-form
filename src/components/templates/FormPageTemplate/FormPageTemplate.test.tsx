import { describe } from "vitest";

describe("FormPageTemplate", () => {
  it("should show active step component", () => {

  })
})
