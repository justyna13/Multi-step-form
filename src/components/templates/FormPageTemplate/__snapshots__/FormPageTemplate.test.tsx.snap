// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FormPageTemplate > should match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="form-card"
        data-testid="test"
      >
        <div
          class="form-wrapper"
        >
          <div
            class="form-navigation"
            data-testid="multiStepForm-navigation"
          >
            <div
              class="form-step "
              data-testid="form-step-1"
            >
              <div
                class="form-step__number"
              >
                1
              </div>
              <div>
                <span
                  class="form-step__info"
                >
                  Step 
                  1
                </span>
                <br />
                <span
                  class="form-step__title"
                >
                  Your info
                </span>
              </div>
            </div>
            <div
              class="form-step active"
              data-testid="form-step-2"
            >
              <div
                class="form-step__number"
              >
                2
              </div>
              <div>
                <span
                  class="form-step__info"
                >
                  Step 
                  2
                </span>
                <br />
                <span
                  class="form-step__title"
                >
                  Select plan
                </span>
              </div>
            </div>
            <div
              class="form-step "
              data-testid="form-step-3"
            >
              <div
                class="form-step__number"
              >
                3
              </div>
              <div>
                <span
                  class="form-step__info"
                >
                  Step 
                  3
                </span>
                <br />
                <span
                  class="form-step__title"
                >
                  Add-ons
                </span>
              </div>
            </div>
            <div
              class="form-step "
              data-testid="form-step-4"
            >
              <div
                class="form-step__number"
              >
                4
              </div>
              <div>
                <span
                  class="form-step__info"
                >
                  Step 
                  4
                </span>
                <br />
                <span
                  class="form-step__title"
                >
                  Summary
                </span>
              </div>
            </div>
          </div>
          <div
            class="form-inner"
          >
            <div
              class="form-confirmation"
              data-testuid="form-confirmation"
            >
              <svg
                height="80"
                viewBox="0 0 80 80"
                width="80"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                >
                  <circle
                    cx="40"
                    cy="40"
                    fill="#F9818E"
                    r="40"
                  />
                  <path
                    d="M48.464 79.167c.768-.15 1.53-.321 2.288-.515a40.04 40.04 0 0 0 3.794-1.266 40.043 40.043 0 0 0 3.657-1.63 40.046 40.046 0 0 0 12.463-9.898A40.063 40.063 0 0 0 78.3 51.89c.338-1.117.627-2.249.867-3.391L55.374 24.698a21.6 21.6 0 0 0-15.332-6.365 21.629 21.629 0 0 0-15.344 6.365c-8.486 8.489-8.486 22.205 0 30.694l23.766 23.775Z"
                    fill="#E96170"
                  />
                  <path
                    d="M40.003 18.333a21.58 21.58 0 0 1 15.31 6.351c8.471 8.471 8.471 22.158 0 30.63-8.47 8.47-22.156 8.47-30.627 0-8.47-8.472-8.47-22.159 0-30.63a21.594 21.594 0 0 1 15.317-6.35Zm9.865 15c-.316.028-.622.15-.872.344l-12.168 9.13-5.641-5.642c-1.224-1.275-3.63 1.132-2.356 2.356l6.663 6.663c.56.56 1.539.63 2.173.156l13.326-9.995c1.122-.816.43-2.993-.956-3.013a1.666 1.666 0 0 0-.17 0Z"
                    fill="#FFF"
                  />
                </g>
              </svg>
              <h2>
                Thank you!
              </h2>
              <p>
                Thanks for confirming your subscription! We hope you have fun using our platform. If you ever need support, please feel free to email us at support@loremgaming.com.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="form-card"
      data-testid="test"
    >
      <div
        class="form-wrapper"
      >
        <div
          class="form-navigation"
          data-testid="multiStepForm-navigation"
        >
          <div
            class="form-step "
            data-testid="form-step-1"
          >
            <div
              class="form-step__number"
            >
              1
            </div>
            <div>
              <span
                class="form-step__info"
              >
                Step 
                1
              </span>
              <br />
              <span
                class="form-step__title"
              >
                Your info
              </span>
            </div>
          </div>
          <div
            class="form-step active"
            data-testid="form-step-2"
          >
            <div
              class="form-step__number"
            >
              2
            </div>
            <div>
              <span
                class="form-step__info"
              >
                Step 
                2
              </span>
              <br />
              <span
                class="form-step__title"
              >
                Select plan
              </span>
            </div>
          </div>
          <div
            class="form-step "
            data-testid="form-step-3"
          >
            <div
              class="form-step__number"
            >
              3
            </div>
            <div>
              <span
                class="form-step__info"
              >
                Step 
                3
              </span>
              <br />
              <span
                class="form-step__title"
              >
                Add-ons
              </span>
            </div>
          </div>
          <div
            class="form-step "
            data-testid="form-step-4"
          >
            <div
              class="form-step__number"
            >
              4
            </div>
            <div>
              <span
                class="form-step__info"
              >
                Step 
                4
              </span>
              <br />
              <span
                class="form-step__title"
              >
                Summary
              </span>
            </div>
          </div>
        </div>
        <div
          class="form-inner"
        >
          <div
            class="form-confirmation"
            data-testuid="form-confirmation"
          >
            <svg
              height="80"
              viewBox="0 0 80 80"
              width="80"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
              >
                <circle
                  cx="40"
                  cy="40"
                  fill="#F9818E"
                  r="40"
                />
                <path
                  d="M48.464 79.167c.768-.15 1.53-.321 2.288-.515a40.04 40.04 0 0 0 3.794-1.266 40.043 40.043 0 0 0 3.657-1.63 40.046 40.046 0 0 0 12.463-9.898A40.063 40.063 0 0 0 78.3 51.89c.338-1.117.627-2.249.867-3.391L55.374 24.698a21.6 21.6 0 0 0-15.332-6.365 21.629 21.629 0 0 0-15.344 6.365c-8.486 8.489-8.486 22.205 0 30.694l23.766 23.775Z"
                  fill="#E96170"
                />
                <path
                  d="M40.003 18.333a21.58 21.58 0 0 1 15.31 6.351c8.471 8.471 8.471 22.158 0 30.63-8.47 8.47-22.156 8.47-30.627 0-8.47-8.472-8.47-22.159 0-30.63a21.594 21.594 0 0 1 15.317-6.35Zm9.865 15c-.316.028-.622.15-.872.344l-12.168 9.13-5.641-5.642c-1.224-1.275-3.63 1.132-2.356 2.356l6.663 6.663c.56.56 1.539.63 2.173.156l13.326-9.995c1.122-.816.43-2.993-.956-3.013a1.666 1.666 0 0 0-.17 0Z"
                  fill="#FFF"
                />
              </g>
            </svg>
            <h2>
              Thank you!
            </h2>
            <p>
              Thanks for confirming your subscription! We hope you have fun using our platform. If you ever need support, please feel free to email us at support@loremgaming.com.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
